default.nix

```diff
@@ -17,7 +17,7 @@ buildGoModule rec {
   version = "0.8.11";
   src = ./.;
   proxyVendor = true; # Fixes: cannot query module due to -mod=vendor running make install
-  vendorHash = "sha256-K1iMGDetMcC1qi+IQ9TkPSMhKMDv4TvOLzVs5Fj3eZo=";
+  vendorHash = "sha256-zv2tJOf1Kjbh3omSctYeY3UBeH8QW1p6LVbvV1yq9QQ=";
```

Comment:
This should have been done in #109 when we upgraded to 1.23 but didn't see the issue until we depended on it.